popSimManu.tex

```diff
@@ -475,6 +478,9 @@ \subsection*{Use case: comparing methods of demographic inference}
 Instead, many methods work by matching coalescence time distributions,
 and a single-population model with varying population size can match any coalescence time distribution
 (in which case coalescence rate is the inverse of the effective size).
+%KEL: I think the sentence above is a bit confusing--maybe rephrase as "Instead, many methods work by matching coalescence time distributions,
+%and a single-population model with varying population size can be fit to a coalescence time distribution in data generated from a more complex model including migrattion.
+% In this case, the coalescence rate is the inverse of the effective population size.
```

Comment:
I support rewording the sentence, but this new one doesn't say the same thing: the current wording says something stronger than "can be fit". How about
> Instead, many methods work by matching coalescence time distributions, and a single-population model with varying population size can be fit to a coalescence time distribution in data generated from a more complex model including migration. In this case, the coalescence time distributions will exactly match, and the coalescence rate is the inverse of the effective population size.

Reply:
is "in data" right here? sounds maybe a bit strange to me. perhaps "using data"?

Reply:
Good point, that's a bad sentence. How about:

> a single-population model with varying population size can be fit to data generated from a more complex model including migration.